editor/src/document/layer_panel.rs

```diff
@@ -21,8 +21,7 @@ impl LayerMetadata {
 }
 
 pub fn layer_panel_entry(layer_metadata: &LayerMetadata, transform: DAffine2, layer: &Layer, path: Vec<LayerId>) -> LayerPanelEntry {
-	let layer_type: LayerDataTypeDiscriminant = (&layer.data).into();
-	let name = layer.name.clone().unwrap_or_else(|| format!("Unnamed {}", layer_type));
+	let name = layer.name.clone().unwrap_or_else(|| String::from(""));
```

Comment:
Can be simplified to `.unwrap_or(String::new()))`

Reply:
Thanks, I suspected there ought to be a better way but didn't know what.

Reply:
Wait, no, Clippy asks me to switch it back to avoid the greedy call.

Reply:
even better: `unwrap_or_default()`

Reply:
Ah, yes, of course!